inc/order_item.class.php

```diff
@@ -1137,9 +1137,16 @@ public function getItems($rand, $data_ref, $plugin_order_orders_id, $numref, $ca
                 "$table.plugin_order_ordertaxes_id",
                 "$table.price_ati"
             ],
-            'FROM' => [$table, $table_ref],
+            'FROM' => $table,
+            'LEFT JOIN' => [
```

Comment:
I'm not sure if LEFT is necessary?
```suggestion
            'JOIN' => [
```

The counter query will most likely need to be changed in the same way (line 1107).